apps/music/js/app.js

```diff
@@ -258,6 +258,7 @@ function setHeaderTitle(title) {
 
 function setBackButtonHidden(hidden) {
   header.els.actionButton.style.visibility = hidden ? 'hidden' : 'visible';
+  scanProgress.hidden = !hidden;
```

Comment:
Couldn't this inadvertently end up showing `scanProgress` when there's no progress to show?